virtool/fake/factory.py

```diff
@@ -232,6 +233,10 @@ async def load_test_case_from_yml(
 
     settings = Settings(**await get(app["db"]))
 
+    # settings2 = await get_data_from_app(app).settings.get_all()
+
+    # Not sure why settings2 is None when it should be the same as settings?
+
```

Comment:
I was trying to use get_all instead of get here, but I'm not sure why settings2 is None and not the same as settings as I would expect.

Reply:
Is this still a problem? I don't quite understand.

Reply:
Fixed, the issue was that {} was being returned when no settings were found. Now get_all is returning Settings() instead.